signals/helpers.py

```diff
@@ -0,0 +1,16 @@
+import os.path
+import glob
+
+
+def recursively_find_parent_containing_file(current_path, search_files):
```

Comment:
Yes, it goes up to the root.  I'm using the same technique to determine where to write the settings file currently.  Unless we want to add another required command line parameter, I'm not sure of another way.